### ISSUE TITLE
Electron Forge docs subpage shows wrong content and has broken CSS

### ISSUE THREAD

**Issue by nikwen** [electron-forge/electron-forge-docs#208]
URL that is broken: https://js.electronforge.io/interfaces/_electron_forge_maker_squirrel.InternalOptions.Options.html

Tested on latest Firefox and Safari.

Page looks like this:

<details><summary>Screenshot</summary>
<p>

![image](https://github.com/user-attachments/assets/95583e4c-7767-47ed-9ee8-3938de9aed9d)

</p>
</details>

**Comment by nikwen** [comment]
Hm, I just noticed that it's not just the CSS that's broken. It also shows a different interface than expected (`@electron/packager` instead of MakerSquirrel).

~~I got this link from my browser history. It could be the case that the page simply no longer exists. I don't see it linked from the site. I'll close this for now.~~

**Comment by nikwen** [comment]
Not true. It's linked [here](https://js.electronforge.io/modules/_electron_forge_maker_squirrel.html#default).

**Comment by Drishti84** [comment]
hello @nikwen can you please assign this issue to me ?? and tell me the link to the codebase
